rustup.hcl

```diff
@@ -50,6 +50,10 @@ on "unpack" {
   run { cmd = "/bin/ln" args = [ "rustup", "rustfmt" ] }
 }
 
+on install {
```

Comment:
Is this a good idea? What if you don't want stable?

Reply:
1. rustup's normal behaviour is to have done `default nightly`; our Hermit hack didn't do it because it's a hack.
2. Users can still `rustup default nightly` or whatever they wish; it will all work as normal!